tutorial/README.md

```diff
@@ -0,0 +1,120 @@
+# Data preparatation
+
+Viv supports a subset of the files generated from the `bioformats2raw` + `raw2ometiff` pipeline, described 
+[here](https://www.glencoesoftware.com/blog/2019/12/09/converting-whole-slide-images-to-OME-TIFF.html). 
+This guide will demonstrate how to generate a pyramidal Zarr or OME-TIFF with Bio-Formats that can be viewed with 
+[Avivator](http://avivator.gehlenborglab.org) via HTTP.
+
+
+### Getting Started
+
+This tutorial requires Bio-Formats [`bioformats2raw`](https://github.com/glencoesoftware/bioformats2raw) and 
+[`raw2ometiff`](https://github.com/glencoesoftware/raw2ometiff) command-line tools. It's easiest to install 
+these tools using `conda`, but binaries also are available for download on the corresponding github repositories.
```

Comment:
I would expect most people to know about conda, but just in case it may be good to add a link
```suggestion
these tools using [`conda`](https://docs.conda.io/projects/conda/en/latest/user-guide/install/), but binaries also are available for download on the corresponding github repositories.
```